Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.11-slim
+LABEL maintainer="jacktyler2391@gmail.com"
+
+ENV PYTHOUNNBUFFERED 1
+
+WORKDIR /cinema
+
+COPY requirements.txt requirements.txt
```

Comment:
Ensure that the `requirements.txt` file exists in the context directory. The `COPY` command is correct if the file is present.